### PR TITLE
ICML RML PC Console without blind submissions and meta reviews.

### PR DESCRIPTION
**Pull request by pmandler**
There is still area chair code left in this from UAI.

### CODE REVIEW

File: venues/ICML.cc/2018/RML/webfield/programchairWebfield.js [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+        byReviewers: reviewerMap
+      };
+    })
+    .fail(function(error) {
+      displayError();
+      return null;
+    });
+};
+
+var getAreaChairGroups = function(noteNumbers) {
```
Comment (melisabok) [inline venues/ICML.cc/2018/RML/webfield/programchairWebfield.js]: We can remove this function

File: venues/ICML.cc/2018/RML/webfield/programchairWebfield.js [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+
+// Kick the whole thing off
+var headerLoaded = $.Deferred();
+displayHeader(headerLoaded);
+
+$.ajaxSetup({
+  contentType: 'application/json; charset=utf-8'
+});
+
+controller.addHandler('areachairs', {
```
Comment (melisabok) [inline venues/ICML.cc/2018/RML/webfield/programchairWebfield.js]: remove this code?

File: venues/ICML.cc/2018/RML/webfield/programchairWebfield.js (hunk 1 above)
Comment (pmandler) [reply venues/ICML.cc/2018/RML/webfield/programchairWebfield.js]: When I do it gets stuck "loading".  It creates areaChairGroups.byNote that is used elsewhere. When I take out those references too it still gets stuck, but I don't see the error.

Comment (melisabok) [reply venues/ICML.cc/2018/RML/webfield/programchairWebfield.js]: There is console in the browser to see the errors

File: venues/ICML.cc/2018/RML/webfield/programchairWebfield.js [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+    var user = pl.user;
+
+    getNotes()
+    .then(function(notes) {
+      var noteNumbers = _.map(notes, function(note) { return note.number; });
+      return $.when(
+        notes,
+        getOfficialReviews(noteNumbers),
+        getReviewerGroups(noteNumbers),
+        getAreaChairGroups(noteNumbers),
```
Comment (melisabok) [inline venues/ICML.cc/2018/RML/webfield/programchairWebfield.js]: You need to remove this call, and the areaChairGroups variable and ...

File: venues/ICML.cc/2018/RML/webfield/programchairWebfield.js (hunk 2 above)
Comment (melisabok) [reply venues/ICML.cc/2018/RML/webfield/programchairWebfield.js]: I think this is a token used by the controller, we could rename it to 'programChairsConsole'?

**Comment by melisabok** [comment]
Great! this is going to be very helpful for other workshops. 

There are only a few lines to remove and it is ready to merge.

**Pull request merged**